scripts/test_env.py

```diff
@@ -36,20 +37,32 @@ def __init__(self, args: argparse.Namespace):
                           Please provide them in the `{args.root_dir}/configs/peer` directory")
             sys.exit(1)
         copy_or_prompt_build_bin("iroha", args.root_dir, peers_dir)
-
+        
         self.peers = [_Peer(args, i) for i in range(args.n_peers)]
 
         os.environ["IROHA2_CONFIG_PATH"] = str(peers_dir.joinpath("config.json"))
         os.environ["IROHA2_GENESIS_PATH"] = str(peers_dir.joinpath("genesis.json"))
         os.environ["IROHA_GENESIS_ACCOUNT_PUBLIC_KEY"] = self.peers[0].public_key
         os.environ["IROHA_GENESIS_ACCOUNT_PRIVATE_KEY"] = self.peers[0].private_key
+        
+        saved = self.peers[0].public_key
+        if is_startup_benchmark:
+            self.peers[0].public_key = "ed012072AED89EDB3CAA5EA203C6CED29A23CB387C9F5C48DB025E524DC508E18A8E22"
+            self.peers[1].public_key = "ed01200B06FCBBD0724DC11D25FC1A88B110B756712462DD6C0D762411E1A98B295AC8"
+            self.peers[2].public_key = "ed01203D922F3E7834F7AF20DAE2DE1AB2F48B4CF8FE95C62D372897692A64143F82C7"
+            self.peers[3].public_key = "ed01208BC55A04A1FC36A15DC63C59E8979FB7080E0937DDE11F2C3F783E9E167ABF4E"
```

Comment:
As I can see we generate private keys randomly (without seed) using `kagami`.
So why do you reassign them here and with hardcoded values? Especially when `private_key` is not changed

Reply:
It was to get the node to accept the saved benchmark blockchain. It's not necessary now so I have removed it in the latest version of this PR.